src/components/slider/test-gesture-config.ts

```diff
@@ -0,0 +1,39 @@
+import {Injectable} from '@angular/core';
+import {MdGestureConfig} from '@angular2-material/core/gestures/MdGestureConfig';
+
+/**
+ * An extension of MdGestureConfig that exposes the underlying HammerManager instances.
+ * Tests can use these instances to emit fake gesture events.
+ */
+@Injectable()
+export class TestGestureConfig extends MdGestureConfig {
+  /**
+   * A map of Hammer instances to element.
+   * Used to emit events over instances for an element.
+   */
+  hammerInstances: Map<HTMLElement, HammerManager[]> = new Map<HTMLElement, HammerManager[]>();
+
+  /**
+   * Create a mapping of Hammer instances to element so that events can be emitted during testing.
+   */
+  buildHammer(element: HTMLElement) {
+    let mc = super.buildHammer(element);
+
+    if (!this.hammerInstances.get(element)) {
+      this.hammerInstances.set(element, [mc]);
+    } else {
+      this.hammerInstances.get(element).push(mc);
+    }
```

Comment:
Nitpick: any way you can flip this conditional?  Trying to scan "if not / else" is a bit harder than "if / else".

```ts
if (this.hammerInstances.get(element)) {
  this.hammerInstances.get(element).push(mc);  
} else {
  this.hammerInstances.set(element, [mc]);
}
```